src/ingest_llm_as/observability/langfuse_client.py

```
@@ -6,207 +6,120 @@
+        return self.is_available()
+
     def create_trace(
         self,
         name: str,
-        session_id: Optional[str] = None,
-        user_id: Optional[str] = None,
         metadata: Optional[Dict[str, Any]] = None,
-        tags: Optional[list] = None,
-        input_data: Optional[Any] = None,
-        output_data: Optional[Any] = None,
+        tags: Optional[List[str]] = None,
```

Ruff (0.13.3)

Unused method argument: `tags`

(ARG002)